Documentation/ApiOverview/Authentication/AuthenticationService/Index.rst

```diff
@@ -161,11 +161,8 @@ processLoginDataBE, processLoginDataFE
   :ref:`login data <authentication-data>`.
 
   The method to implement is :php:`processLoginData()`.
-  It receives as argument the login data and the password
-  transmission strategy (which corresponds to the login security
-  level, where only 'normal' can be used.
-  It returns the boolean value :php:`true`,
-  when the login data has been successfully processed .
+  It receives as the only argument the login data and returns the boolean
```

Comment:
We need a `..  versionchanged::` https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/Reference/ReStructuredText/Content/Versions.html#versionchanged Infomation here